### PR TITLE
add OPV

### PR DESCRIPTION
**Pull request by kjappelbaum**

### CODE REVIEW

**Comment by kjappelbaum** [comment]
Screenshot 2023-10-18 at 21 57 22

not a lot of data, but I hope it adds some value

File: data/tabular/opv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      units: mA/cm^2
+      type: continuous
+      names:
+          - noun: short-circuit current density of tested devices
+          - noun: short-circuit current density
+      uris:
+      significant_digits: 2
+    - id: FF
+      description: Fill factor of devices with this polymer
+      units: ''
```
Comment (MicPie) [inline data/tabular/opv/meta.yaml:44]: dimensionless?

**Review by MicPie — APPROVED** [review]

File: data/tabular/opv/meta.yaml (hunk 1 above)
Comment (MicPie) [reply data/tabular/opv/meta.yaml:44]: If, I would exclude it from the general templates, as they always assume a unit.

File: data/tabular/opv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      Answer: {#The open-circuit voltage is |The Voc is!}{Voc#} {Voc__units}.
+    - |-
+      Question: What is the {Jsc__names__noun} of a {#non-fullerene|PC71BM|PCBM!} {#organic photovoltaics|OPV|organic solar cell|organic photovoltaics (OPV)!} device with a donor polymer with monomer SMILES {SMILES#} and {#Mw|weight-average molecular weight|weight-average molecular weight (Mw)!} {Mw#} g/mol and {#PDI|polydispersity index|polydispersity index (PDI)!} of {PDI#}?
+      Answer: {#The short-circuit current density is |The Jsc is!}{Jsc#} {Jsc__units}.
+    - |-
+      Question: What is the {FF__names__noun} of a {#non-fullerene|PC71BM|PCBM!} {#organic photovoltaics|OPV|organic solar cell|organic photovoltaics (OPV)!} device with a donor polymer with monomer SMILES {SMILES#} and {#Mw|weight-average molecular weight|weight-average molecular weight (Mw)!} {Mw#} g/mol and {#PDI|polydispersity index|polydispersity index (PDI)!} of {PDI#}?
+      Answer: {#The fill factor is |The FF is !}{FF#} {FF__units}.
```
Comment (kjappelbaum) [inline data/tabular/opv/meta.yaml:130]:
```suggestion
      Answer: {#The fill factor is |The FF is !}{FF#}.
```

**Pull request merged**